app/models/alchemy/element_page.rb

```diff
@@ -0,0 +1,7 @@
+module Alchemy
+  class ElementPage
+    def self.table_name
+      [Element.table_name, Page.table_name].join('_')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.